modules/AWS/eks/nodegroup_launch_template/locals.tf

```diff
@@ -25,6 +25,7 @@ locals {
   templates_all = var.osquery_secret_name != "" ? concat(tolist(local.templates), ["osquery/osquery.sh.tpl"]) : local.templates
 
   user_content              = [ for tpl in local.templates_all : templatefile("${path.module}/templates/${tpl}", {
+    cluster_name                    = var.cluster_name
```

Comment:
Is this a new required value?

Reply:
Thanks for catching that! It's unused indeed.